src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -485,27 +486,42 @@ fun sendInfotrygdOppdatering(
     val personNrPasient = receivedSykmelding.personNrPasient
     val signaturDato = receivedSykmelding.sykmelding.signaturDato.toLocalDate()
     val tssid = receivedSykmelding.tssid
+    val uforegrad = when {
+        perioder.first().gradertSykmelding != null -> perioder.first().gradertSykmelding.sykmeldingsgrad.toBigInteger()
+        perioder.first().aktivitetIkkeMulig != null -> 100.toBigInteger()
+        else -> 0.toBigInteger()
+    }
     val sha256StringMedArbeidsgiver = sha256hashstring(createInfotrygdBlokk(
             itfh, perioder.first(), personNrPasient, signaturDato,
-            behandlerKode, tssid, loggingMeta, navKontorNr, findarbeidsKategori(itfh.healthInformation.arbeidsgiver?.navnArbeidsgiver))
-    )
+            behandlerKode, tssid, loggingMeta, navKontorNr,
+            findarbeidsKategori(itfh.healthInformation.arbeidsgiver?.navnArbeidsgiver), uforegrad))
     val sha256StringUtenArbeidsgiver = sha256hashstring(createInfotrygdBlokk(
             itfh, perioder.first(), personNrPasient, signaturDato,
-            behandlerKode, tssid, loggingMeta, navKontorNr, "")
+            behandlerKode, tssid, loggingMeta, navKontorNr, "", uforegrad)
     )
+    val sha256StringUtenUforegrad = sha256hashstring(createInfotrygdBlokk(
+            itfh, perioder.first(), personNrPasient, signaturDato,
+            behandlerKode, tssid, loggingMeta, navKontorNr,
+            findarbeidsKategori(itfh.healthInformation.arbeidsgiver?.navnArbeidsgiver), 0.toBigInteger()))
 
     try {
         val redisSha256StringMedArbeidsgiver = jedis.get(sha256StringMedArbeidsgiver)
         val redisSha256StringUtenArbeidsgiver = jedis.get(sha256StringUtenArbeidsgiver)
+        val redissha256StringUtenUforegrad = jedis.get(sha256StringUtenUforegrad)
```

Comment:
Det slår meg nå at hvis redis er nede, eller hvis redisen wipes, så gjør vi funksjonelle feil - dvs at vi sender et annet resultat til infotrygd enn vi ville ha gjort hvis redis var oppe/oppdatert. Jeg tror vi bør bytte ut redis med en database, for slik vi bruker redis nå tåler vi ikke egentlig at den ikke er der, og da tror jeg usecaset vårt stemmer bedre med en database. Dessverre :(

Reply:
Jepp, database it is